src/Rapidez.php

```diff
@@ -100,6 +102,38 @@ public function setStore(Store|array|callable|int|string $store): void
         config()->set('rapidez.root_category_id', $store['root_category_id']);
     }
 
+    public function getTaxTable(): array
+    {
+        return Cache::rememberForever('tax_'.config('rapidez.store'), function () {
+            $table = [];
+            $rates = DB::table('tax_class')
+                ->join('tax_calculation', 'tax_calculation.product_tax_class_id', '=', 'tax_class.class_id')
+                ->join('tax_calculation_rate', 'tax_calculation_rate.tax_calculation_rate_id', '=', 'tax_calculation.tax_calculation_rate_id')
+                ->where('tax_country_id', Rapidez::config('tax/defaults/country', 'US'))
+                ->where('tax_region_id', Rapidez::config('tax/defaults/region', 0))
+                ->whereIn('tax_postcode', [null, '*', Rapidez::config('tax/default/postcode', null)])
```

Comment:
Do we want to have the full tax calculation table here instead? Right now this will only base your tax rates off of the country the shop has set as its default, which might be problematic for more international shops. I'm not well-versed enough in tax law to know how this is supposed to work 😅

Reply:
Magento does this: https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/Tax/Model/ResourceModel/Calculation.php#L267

Basically, just calculates the rate on request in the backend by combining all the factors there (then caching it). The entire table isn't really exposed anywhere that way.

I'm not sure what the best approach for rapidez would be. Probably not exactly that?

Reply:
That does not seem to be the bit of code responsible for the postcode it should use, that has already been placed inside of $request.
What i'm most interested in is how Magento would deal with prices on the catalog, as magento caches every page as well thus this bit of backend code would not be able to influence the prices directly.

Maybe we can place the default tax amount in the frontend, and once another postcode is added send an api request and update the tax amount based on what it responds with?